src/SupportServiceProvider.php

```diff
@@ -11,6 +13,15 @@ public function configureTipoffPackage(TipoffPackage $package): void
             ->hasConfigFile('tipoff');
     }
 
+    public function bootingPackage()
+    {
+        parent::bootingPackage();
+
+        Blade::directive('tipoffMoney', function ($amount) {
```

Comment:
@drewroberts - note the `tipoff` prefix on the directive registration.  Seemed a bit aggressive to register a common term without some prefix.